data/ShubhamChikane99.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Shubham Chikane",
+  "type": "personal",
+  "bio": "Open Source, DevRel, DevOps ",
+ "tags": [
+    "Open Source",
+    "C++",
+    "JAVA",
+    "HTML",
+     "CSS",
+    "DevOps",
+    "DevRel"
+  ],
+ "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/shubhamchikane_" },
+    { "icon": "FaGithub", "url": "https://github.com/ShubhamChikane99" }
+   ],
+"links": [
+    {
+      "group": "Primary Socials",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/shubhamchikane_",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+   {
+      "group": "Primary Socials",
+      "name": "LinkedIn: Connect with me ",
+      "url": "https://linkedin.com/in/shubham chikane",
```

Comment:
hey @ShubhamChikane99, your LinkedIn profile doesn't exist. Please add a valid URL. 
✌️

Reply:
It's not a blocker, the fix can come in another PR. merging it now

Thank you